cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamLeaderManager.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.data.stream.service;
+
+import com.google.common.util.concurrent.ListenableFuture;
+import com.google.common.util.concurrent.Service;
+import org.apache.twill.discovery.Discoverable;
+
+/**
+ * Manages Streams leaders. One Stream handler leader is affected to every Stream. The leader can perform any
+ * global processing on the stream, like aggregating the size of the stream as given by each individual stream writers.
```

Comment:
like -> "such as"

such as aggregating the size of the stream as given by an individual stream writer.